perfkitbenchmarker/gcp/gce_network.py

```diff
@@ -66,29 +119,16 @@ def AllowPort(self, vm, port):
     with self._lock:
       firewall_name = ('perfkit-firewall-%s-%d' %
                        (FLAGS.run_uri, port))
-      if firewall_name in self.firewall_names:
+      firewall_rule = GceFirewallRule(firewall_name, self.project, port)
+      if firewall_rule in self.firewall_rules:
```

Comment:
oops - it used to be a list, and I wasn't thinking when changed it to a set - changed back